index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>TriviaGame</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- Compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
+    <!-- MY CSS -->
+    <link rel="stylesheet" type="text/css" media="screen" href="assets/css/style.css" />
+</head>
+<body>
+    <div class="row Header">
+        <div class="col s12 center black white-text">
+            <h1>Trivia Game<h1>
+        </div>
+        <div class="col s12 center black-text">
+            <h2 id="display"></h2>
+        </div>
+    </div>
+    <div class="container row score">
+        <div class="col s12 scoreText"></div>
+    </div>
+    <div class="questions black-text">
+        <div class="container row Q1">
+            <div class="col s12 left">
+                <h5>What is My Favorite Color?</h5>
```

Comment:
try dynamically displaying the questions from your js file. It will be easier to work with and make your code more reusable.